coordinator/src/main/java/org/apache/uniffle/coordinator/strategy/storage/AppBalanceSelectStorageStrategy.java

```diff
@@ -132,4 +76,9 @@ public synchronized RemoteStorageInfo pickStorage(String appId) {
     LOG.warn("No remote storage is available, we will default to the first.");
     return availableRemoteStorageInfo.values().iterator().next();
   }
+
+  @Override
+  public Comparator<Map.Entry<String, RankValue>> getComparator() {
+    return (o1, o2) -> 0;
```

Comment:
I think return null is better, We don't need to sort the paths in this case.

Reply:
Ok, updated.